packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/recent.tsx

```diff
@@ -10,100 +10,48 @@ import { getSavedWorkspaces } from '@deriv/bot-skeleton';
-                        </div>
-                        <div className='load-strategy__recent__files__list'>
-                            {recent_strategies.map((workspace, index) => {
-                                return <RecentWorkspace key={workspace.id} workspace={workspace} index={index} />;
-                            })}
-                        </div>
-                        <DeleteDialog />
-                    </div>
-                </div>
-            </div>
-        );
-    }
+        getSavedWorkspaces().then(recent_strategies => setDashboardStrategies(recent_strategies));
+    }, []);
 
+    if (!dashboard_strategies?.length) return null;
     return (
-        <div className='load-strategy__container'>
-            <div className='load-strategy__recent__empty'>
-                <Icon icon='IcEmptyFolder' className='load-strategy__recent__empty-icon' size={128} />
-                <div className='load-strategy__recent-empty-title'>
-                    <Localize i18n_default_text='You do not have any recent bots' />
-                </div>
-                <div className='load-strategy__recent-empty-description'>
-                    <Localize i18n_default_text='Create one or upload one from your local drive or Google Drive.' />
-                </div>
-                <div className='load-strategy__recent-empty-expand' onClick={toggleExplanationExpand}>
-                    <Localize i18n_default_text="Why can't I see my recent bots?" />
-                </div>
-                <div
-                    className={classnames('load-strategy__recent-empty-explanation', {
-                        'load-strategy__recent-empty-explanation--show': is_explanation_expand,
-                    })}
-                >
-                    <div>
-                        <Localize i18n_default_text="If you've recently used bots but don't see them in this list, it may be because you:" />
+        <div className='load-strategy__container load-strategy__container--has-footer'>
+            <div className='load-strategy__recent'>
+                <div className='load-strategy__recent__files'>
+                    <div className='load-strategy__title'>
+                        <Localize i18n_default_text='Your Bots' />
                     </div>
-                    <ol className='load-strategy__recent-empty-explanation-list'>
-                        {explanation_list.map(content => {
-                            return (
-                                <li key={content}>
-                                    <Localize i18n_default_text={content} />
-                                </li>
-                            );
+                    <div className='load-strategy__recent__files__list'>
+                        <div className='load-strategy__recent-item load-strategy__recent-item__loaded load-strategy__recent-item__loaded--first-child'>
+                            <Text weight='bold'>Name</Text>
```

Comment:
or maybe we can use a map instea dof repeating

Reply:
done ✅